ReactWindows/ReactNative/Modules/Vibration/VibrationModule.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using ReactNative.Bridge;
+using Windows.Phone.Devices.Notification;
+
```

Comment:
Sort (I know, the System namespaces usually come first, but we've done alphabetical everywhere else).